persistence-api/src/main/java/io/stargate/db/schema/Column.java

```diff
@@ -431,6 +459,7 @@ public boolean isUserDefined() {
     private String description;
 
     private Class<?> javaType;
+    private final String marshalTypeName;
```

Comment:
maybe `customTypeName`? to match the native protocol name and because it's used for custom types. 

https://github.com/apache/cassandra/blob/trunk/doc/native_protocol_v4.spec#L597-L598

Reply:
will do in a follow-up PR.

Reply:
As I hinted in another comment this type name is "custom" only from the protocol perspective, since it does not have a unique type ID. From the storage perspective it is a first-class type like `ascii` or `boolean`, AFAIK.

I'll think about cleaning this up in a follow-up PR.